Add initial instructions for cluster role map updating (#19006)
gravitational/teleport: docs/pages/management/admin/trustedclusters.mdx

</Admonition>

> <Details

Reviewer: I'm not sure Step 4 is the best place for this. Should we add these instructions where the `Details` box that this PR removes used to be?

Author: That sounds good to me, have moved it over.

  type="tip"
  title="Update role mappings"
>

> You can update the role mappings by modifying the trusted_cluster.yaml file we created
> earlier to contain the new role mapping.

Reviewer: ```suggestion
You can update the role mappings within the `trusted_cluster` resource we created earlier by modifying the resource definition (`trusted_cluster.yaml`) to indicate the new role mappings in the `role_map` field.
```

> After logging into the leaf cluster:

Reviewer 2: ```suggestion
Then, update the trusted cluster resource on the leaf cluster side:
```

  
</Admonition>

> ## Step 5/6. Updating role mappings

Reviewer 3: - I would say `Update` instead of `Updating` to match the form of the other steps
- This isn't really a required step in the process, it's more of a note that you can make updates. Should we put it under step 4 instead of making it a separate step? (I think so, and I'd also mention something like "starting version x.y.z  you can update role mappings without creating a new trusted cluster)

Author: Will we need the 'starting in version xyz" if the docs are already versioned.
I've added the other changes.

You can update the role mappings by modifying the trusted_cluster.yaml file we created
earlier to contain the new role mapping.

After logging into the leaf cluster:

```code

> $ tctl create --force updated_cluster.yaml

Reviewer 3: ```suggestion
$ tctl create --force trusted_cluster.yaml
```

Either keep the filename the same, or tell the user that you expect them to save updates to a different file. Just changing the name might confuse the reader.
